### DISCUSSION TITLE
Full SHA of a commit in repository copying is no longer working in Android with desktop view

### DISCUSSION THREAD

**Discussion by ghost** [community/community#64069] [Repositories]
### Select Topic Area

Bug

### Body

Full SHA of a commit in repository copying is no longer working in Android with desktop view.

**Comment by ayan759619** [comment]
IF you're encountering an issue with copying the full SHA of a commit in an Android repository while using the desktop view. The SHA (Secure Hash Algorithm) is a unique identifier for each commit in a Git repository. If copying the full SHA is not working as expected, you might want to consider the following troubleshooting steps:

Check the Interface: Make sure you're using the desktop view on your Android device. Sometimes, mobile views might have limitations in terms of functionality.

User Interface (UI) Elements: Ensure that you're interacting with the correct UI element to copy the SHA. Usually, the SHA is a clickable link or a button/icon associated with each commit. If you're tapping on the wrong part of the UI, it might not trigger the copy action.

Browser Compatibility: Different browsers can behave differently on mobile devices. Try using different browsers to see if the issue persists. Chrome, Firefox, and other popular browsers might handle desktop view differently.

Browser Settings: In the browser's settings, look for options related to "Desktop Site" or "Request Desktop Site." Enabling this setting forces the website to display the desktop version, which might improve compatibility.

Zoom and Page Scaling: Sometimes, issues can occur due to zoom or page scaling settings. Make sure the page is not overly zoomed in or scaled out, as this can affect how you interact with UI elements.

Cache and Cookies: Clear your browser's cache and cookies. Cached data can sometimes interfere with the proper functioning of a website.

Responsive Design: Many modern websites use responsive design, which adapts the layout to the device's screen size. However, sometimes this can lead to unexpected behavior. You might need to zoom out or scroll horizontally to access certain UI elements.

User Agent Switcher: There are browser extensions available that allow you to change your user agent to mimic a desktop browser. This might help you access the full functionality of the desktop view.

Git Client Apps: If the issue persists, consider using Git client apps for Android. These apps are designed to provide a more user-friendly interface for interacting with Git repositories and might offer better compatibility.

If none of these steps resolve your issue, it's possible that there could be a specific problem with the website's implementation or a more unique compatibility issue with your Android device or browser. In such cases, you might want to consider reaching out to the website's support or community forums for more targeted assistance.

**Reply by ghost** [reply]
I use Chrome Android always.Tried clearing cache. Also changed os version Android 10 and Android 13 .still same .was working long time before .Dead after the github site updates

**Reply by ghost** [reply]
Recently confirmed that it's not working for other friends.

**Comment by bronglil** [comment]
@dib12  Hey Yes, you are correct. The full SHA of a commit in repository copying is no longer working in Android with desktop view. This is a known issue and the developers are working on a fix.

In the meantime, you can use the following workaround:

Right click on the SHA (this will select it)
Press Command+C or CTRL+C
This will copy the full SHA to your clipboard. You can then paste it into another application, such as a text editor or email client.

I hope this helps!

Here are some additional things you can do:

Report the issue to the GitHub Desktop developers so they can fix it as soon as possible.
Use the web version of GitHub until the issue is fixed.
Use a different Git client, such as Fork.

I hope this will help you.

**Reply by ghost** [reply]
Thank you so much.
So this is not the right place where developer can actually see the bug.
Should I close the discussion?
